src/relay/op/nn/pooling.cc

```diff
@@ -720,5 +720,257 @@ RELAY_REGISTER_OP("nn.avg_pool2d_grad")
 .set_attr<FTVMCompute>("FTVMCompute", Pool2DGradCompute<AvgPool2DAttrs, topi::nn::kAvgPool>);
 
 
+// relay.nn.max_pool3d & relay.nn.avg_pool3d
+TVM_REGISTER_NODE_TYPE(MaxPool3DAttrs);
+TVM_REGISTER_NODE_TYPE(AvgPool3DAttrs);
+
+template <typename T>
+Array<Array<Layout> > Pool3DInferCorrectLayout(
```

Comment:
I think this function can be unified for 2d and 3d.

Reply:
ok.